src/ethereum_clis/clis/geth.py

```diff
@@ -290,26 +368,71 @@ def consume_blockchain_test(
                 f"Unexpected exit code:\n{' '.join(command)}\n\n Error:\n{result.stderr}"
             )
 
-    @cache  # noqa
-    def consume_state_test_file(
+    # ------------------------------------ eof test -----------------------------------------------
+    def extract_int_from_subprocess_string(
+        self, s: str, substring_of_interest_prefix: str
+    ) -> int | None:
```

Comment:
```suggestion
    @staticmethod
    def extract_int_from_subprocess_string(
        s: str, substring_of_interest_prefix: str
    ) -> int | None:
```
This should be static since it doesn't access any instance properties, then you can call it with `GethFixtureConsumer.extract_int_from_subprocess_string`.